.circleci/config.yml

```diff
@@ -24,6 +24,7 @@ jobs:
             fi
       - run: make check-go-mod
       - run: make errcheck
+      - run: make docs
```

Comment:
Maybe it could not be added in CI. It will change the documents if something updated.

Reply:
OK, I have delete `make docs` , but it keep in vim swap file , not saving  back . I am sorry for not re-review again

Reply:
Never mind, but it seems to still stay in the `.circleci/config.yml`.. Please make sure your current workspace, you can delete the `.swp` file and edit again.

Reply:
It is so weird , I have checked 'git diff' logs  before go 'git commit' . Never mind , I will do correct it in other way . Thank you ~~

Reply:
OK ， I have done it  @jojohappy